person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  reader_atribute :id
+  accessor_atribute :name, age
```

Comment:
- [ ] Well done so far working on your application's Classes. However, for the attributes syntax, it should be `attr_accessor` and `attr_reader`

Reply:
### Thank you for your attention @MahmoudBakr23